pyproject.toml

```diff
@@ -52,6 +52,7 @@ classifiers = [
 ]
 dependencies = [
   "pyyaml~=6.0",
+  "rich~=14.0",
```

Comment:
This wasn’t an issue before, and tests passed because the CI workflow environment installs all development dependencies during testing. It appears that rich is included as a subdependency of those.

However, this is problematic because users installing the package do not install development dependencies by default. To prevent hidden dependency issues, we should configure the CI to run tests without installing development dependencies.